modisco

```diff
@@ -78,6 +81,61 @@ The options are as follows:
 )
 meme_parser.add_argument("-o", "--output", type=str, help="The path to the output file.")
 
+seqlet_bed_parser = subparsers.add_parser("seqlet-bed", help="Create a BED file of seqlets from a modisco results file.")
+seqlet_bed_parser.add_argument("-i", "--h5py", type=str, required=True,
+	help="An HDF5 file containing the output from modiscolite.")
+seqlet_bed_parser.add_argument("-o", "--output", type=str, required=True,
```

Comment:
Let's get rid of this in favor of stdout

Reply:
Done